lib/consul/index.js

```diff
@@ -104,7 +105,7 @@ var file_modified = function(branch, file, cb) {
       fs.readFile(file_path, {encoding: 'utf8'}, function (err, body) {
         /* istanbul ignore if */
         if (err) return cb('Failed to read key ' + file_path + ' due to ' + err);
-        var body = body ? body.trim() : '';
+        body ? body.trim() : '';
```

Comment:
Is this intentional? Don't you need to assign `body = body ? body.trim() : '';` even if you don't need `var`?